config/passport.js

```diff
@@ -0,0 +1,84 @@
+var GoogleStrategy = require('passport-google-oauth').OAuth2Strategy;
+var LocalStrategy = require('passport-local').Strategy;
+var BearerStrategy = require('passport-http-bearer').Strategy;
+var configAuth = require('./settings');
+var connection = require('./database');
+
+module.exports = function(passport) {
+
+	passport.serializeUser(function(user, done){
```

Comment:
sorry, somehow missed line 7.
indent is required, just please make it consistent with the rest of the code. especially you shouldn't mix the whitespaces (tab vs. space).